sqlalchemy_filters/filters.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 from collections import Iterable, namedtuple
-from inspect import signature
+try:
+    from inspect import signature
+except ImportError:
+    # For python2 capability. NOTE: you must fulfill this requirement
+    # yourself, it is not in requirements.txt
```

Comment:
"# yourself, it is not in package requirements" - there is no requirements.txt for this project, `install_requires` is used instead.

Reply:
+1

Also, how would you feel about adding another section to `extras_require` to support this @iky ? Something like `'python2'`.

https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras